{
    "checksum": "6e1d7628cda63468ed412954ec020f172d2261efd1fd813f08ae210d7593d798",
    "file": "gcalctool-bug-488226.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 2,
    "package": "strongwind",
    "path": "strongwind/0.9-2/examples/gcalctool-bug-488226.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 198,
        "license": "/copyright/license/strongwind/0.9-2/",
        "metric": {
            "size": 252
        },
        "pts_link": "https://tracker.debian.org/pkg/strongwind",
        "sloc": [
            [
                "python",
                1446
            ],
            [
                "makefile",
                18
            ]
        ],
        "suites": [
            "jessie",
            "jessie-kfreebsd",
            "stretch",
            "wheezy"
        ]
    },
    "raw_url": "/data/main/s/strongwind/0.9-2/examples/gcalctool-bug-488226.py",
    "stat": {
        "perms": "rwxr-xr-x",
        "size": 1270,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "0.9-2"
}